tsconfig.release.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "lib": ["es2017"],
```

Comment:
I don't think we will be producing ESM since we explictly set `module: commonjs` below.

I thought we won't be dropping support for Node 8 w/ this change, but as you suggest here it does look like ES20178 might not be fully supported in Node 8 🤔.  I'm still not sure why our release process requires us to install firebase admin (which is the reason why I had to make this change) - I'll go get this cleared up before we do this.

Reply:
Ah I see it now.

Reply:
Offline discussion - we are okay w/ dropping support for Node 8, as both GCF and Firebase CLI prevents users from deploying new Node 8 functions since last december.